packages/common/src/themes/codesandbox-black.js

```diff
@@ -35,7 +35,7 @@ const colors = {
   },
   editor: {
     background: tokens.grays[700],
-    foreground: tokens.white,
+    foreground: tokens.grays[300],
```

Comment:
Should this be `white` again?

Reply:
I changed these values, because of your changes in #3008

Reply:
You changed these in `codesandbox-black.json`, but that file will be deleted and I changed the values in `codesandbox-black.js` which is really used

Reply:
Oh nice!! Really nice.